src/util/metricsAggregator.js

```diff
@@ -0,0 +1,167 @@
+/* eslint-disable */
+const cluster = require('cluster');
+const logger = require('../logger');
+const { Worker, isMainThread } = require('worker_threads');
+const GET_METRICS_REQ = 'rudder-transformer:getMetricsReq';
+const GET_METRICS_RES = 'rudder-transformer:getMetricsRes';
+const AGGREGATE_METRICS_REQ = 'rudder-transformer:aggregateMetricsReq';
+const AGGREGATE_METRICS_RES = 'rudder-transformer:aggregateMetricsRes';
+const RESET_METRICS_REQUEST = 'rudder-transformer:resetMetricsReq';
+const METRICS_AGGREGATOR_PERIODIC_RESET_ENABLED =
```

Comment:
again for readability:

```suggestion
const config = {
  periodicResetEnabled: process.env.METRICS_AGGREGATOR_PERIODIC_RESET_ENABLED === 'true',
  resetIntervalSeconds: parseInt(process.env.METRICS_AGGREGATOR_PERIODIC_RESET_INTERVAL_SECONDS, 10) || 1800
};
```